tools/run_tests/xds_k8s_test_driver/config/url-map.cfg

```diff
@@ -1,6 +1,5 @@
---resource_suffix=interop-psm-url-map
+--resource_prefix=interop-psm-url-map
```

Comment:
Ooops, my fault.

Reply:
No worry. As long as xDS port stays the same, we are good.